en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+# Safety Configuration (Failsafes) 
+
+PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+
+The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+
+## Failsafe Actions {#failsafe_actions}
+
+Each failsafe defines its own set of actions. Some of the more common failsafe actions are:
+
+Action | Description
+--- | --- 
+None/Disabled | No action (the failsafe will be ignored).
+Warning | A warning message will be sent to *QGroundControl*.
+Loiter | The vehicle will enter [Hold mode](../flight_modes/hold.md). For multicopters this means the vehicle will hover, while for fixed/wing the vehicle will circle.
+Return to Land | The vehicle will enter [Return mode](../flight_modes/return.md). Return behaviour can be set in the [Return Home Settings](#return_settings) (below).
+Land at current position |  The vehicle enters [Land mode](../flight_modes/land.md), and lands immediately.
+RC Auto Recovery (CASA Outback Challenge rules) | ?
+Terminate/Flight termination | ?
+Lockdown | ?
+
+
+> **Note** You can only specify the action for the *first* failsafe event. Once a failsafe occurs the system will enter special handling code, such that subsequent failsafe triggers are managed by separate system level and vehicle specific code. This might result in the vehicle being changed to a manual mode so the user can directly manage recovery.
+
+
+## QGroundControl Safety Setup {#qgc_safety_setup}
+
+The *QGroundControl* Safety Setup page is accessed by clicking the *QGroundControl* **Gear** icon (Vehicle Setup - top toolbar) and then **Safety** in the sidebar). This includes the most important failsafe settings (battery, RC loss etc.) and the settings for the return actions *Return* and *Land*.
+
+![Safety Setup (QGC)](../../images/qgc/setup/safety_setup.png)
+
+
+### Low Battery Failsafe
+
+The low battery failsafe is triggered when the battery capacity drops below one (or more warning) level values. 
+
+![Safety - Battery (QGC)](../../images/qgc/setup/safety_battery.png)
+
+The most common configuration is to set the values and action as above (with `Warn > Failsafe > Emergency`).
+With this configuration the failsafe will trigger warning, then return, and finally landing if capacity drops below the respective levels.
+ 
+It is also possible to set the *Failsafe Action* to warn, return, or land. 
+In this case the vehicle will perform the specified action when the *Warn* failsafe level is reached.
```

Comment:
Is this correct? Specifically, is it the Warn level that triggers the failsafe that is specified int eh failsafe action - or is it one of the other thresholds?

Reply:
No, it's at the "Battery Failsafe Level"